tests/test_util.py

```diff
@@ -13,6 +13,7 @@
 import gcs_utils
 import resources
 
+RESOURCES_BUCKET_FMT = '{project_id}-resources'
```

Comment:
@mark-velez project_id is not yet available yet here.

Reply:
This is a format string where `project_id` is a named index..?